indico/modules/events/papers/templates/management/overview.html

```diff
@@ -2,6 +2,7 @@
 
 {% block content %}
     {% set cfp = event.cfp %}
+    {% set tz = event.timezone %}
```

Comment:
I'd use `event.tzinfo` (same behavior, it's just using the tzinfo object instead of the timezone string)